Maple2.Database/Storage/Game/GameStorage.Club.cs

```diff
@@ -1,64 +1,148 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using Maple2.Database.Extensions;
+﻿using Maple2.Database.Extensions;
+using Maple2.Model.Enum;
 using Maple2.Model.Game;
-using Club = Maple2.Model.Game.Club;
-using ClubMember = Maple2.Model.Game.ClubMember;
+using Maple2.Model.Game.Party;
+using Maple2.Tools.Extensions;
+using Z.EntityFramework.Plus;
+using Club = Maple2.Model.Game.Club.Club;
+using ClubMember = Maple2.Model.Game.Club.ClubMember;
 
 namespace Maple2.Database.Storage;
 
 public partial class GameStorage {
     public partial class Request {
         public Club? GetClub(long clubId) {
-            Model.Club? model = Context.Club.Find(clubId);
-            if (model == null) {
+            return Context.Club.Find(clubId);
```

Comment:
This should probably return the fully constructed club. As-is it's missing the members and they always need to be loaded separately.